src/controller/payments.controller.ts

```diff
@@ -520,27 +520,29 @@ export default function (
           return res.status(403).send('Invalid trial token');
         }
 
+        const validTrials = ['pc-cloud-25', 'pc-componentes-25'];
+
         try {
           const payload = jwt.verify(trialToken, config.JWT_SECRET) as { trial?: string };
-          if (!payload.trial || payload.trial !== 'pc-cloud-25') {
+          if (!payload.trial || !validTrials.includes(payload.trial)) {
             throw new Error('Invalid trial token');
           }
-        } catch {
```

Comment:
Why did you remove the 403? This could useful for the mobile app that includes the same offer